topics/fair/tutorials/fair-ena/tutorial.md

```diff
@@ -0,0 +1,305 @@
+> | Read data | Y    | Y    | Y    |
+> | Genome Assembly | N    | Y    | N    |
+> | Transcriptome Assembly | N    | Y    | N    |
+> | Template Sequence | N    | Y    | N    |
+> | Other Analyses | N    | N    | Y    |
+{: .matrix}
+
+As you can see from the Table 1, you can submit some data types only through a certain route. However, read data can be submitted via all three routes. 
+
+To begin with the submission process you will need to have a Webin submission account. You can register an account free of cost through the Webin portal (https://www.ebi.ac.uk/ena/submit/webin/login).
+
+For data submission, ENA provides a detailed guide (https://ena-docs.readthedocs.io/en/latest/submit/general-guide.html), which I recommend you consult for questions specific to your user case. Remember that ENA frequently changes its submission protocols, so you should check the ENA guide for updates.
+
+## ENA Metadata Model
+
+![ENA metadata model](./images/metadata_model.png "ENA metadata model")
+
+The ENA metadata model (Figure 1) represents how metadata for a submission is structured and linked. Different information pertaining to a submission is organised into objects within the metadata model. Therefore, understanding the metadata model will enable you to determine which objects you need to submit. 
+
+### 1. Study
+All submissions require the study object. This object defines study accession, ownership, affiliation and release date. It also binds together all related objects into one cohesive project.
+
+Study object submission generates a BioProject accession (PRJEB*) which can be used for citing data submitted to ENA, and a study accession (ERP*).
+
+### 2. Sample
+Sample object defines metadata about the sequenced biomaterial, e.g. bacteria, virus, etc. One sample should represent one sequencing library. 
+
+To ensure that each sample is registered with sufficient metadata so as to provide enough context for the data to be easily interpreted, a mandatory, recommended and optional set of attributes should be provided for a sample.  Sample checklists have been developed in collaboration with different research communities to ensure that they are relevant and realistic for their context. They differ with the nature of the sample.
+
+A registered sample will receive a BioSample accession (SAMEA*) and ENA sample accession (ERS*).
+
+### 3. Raw reads
+Metadata for raw reads is represented in 'experiment' and 'run' objects, where 'experiment' holds metadata on the sequencing method and 'run' holds metadata about the read files and their location on an FTP server. 
+
+Each read file submission will receive a run accession (ERR*) and an Experiment accession (ERX*). 
+
+# 02. Submitting data
+## 1. Webin login
+Log in to ENA Webin portal https://www.ebi.ac.uk/ena/submit/webin/. If you don't have a login you will have to register for an account. Go ahead and do that - it is free. Click on the 'Register' button and fill in the information requested.  

```

Comment:
```suggestion
[Log in to ENA Webin portal](https://www.ebi.ac.uk/ena/submit/webin/). If you don't have a login you will have to register for an account. Go ahead and do that - it is free. Click on the 'Register' button and fill in the information requested.  
```

(otherwise it won't be a link)